src/doc/rustdoc/src/lints.md

```diff
@@ -70,6 +70,8 @@ This lint **warns by default**. This lint detects when [intra-doc links] from pu
 For example:
 
 ```rust
+#![warn(private_intra_doc_links)]
```

Comment:
Please use the rustdoc:: prefix, I'm hoping to remove the old version eventually.